lab_5_scrapper/scrapper.py

```diff
@@ -204,12 +382,29 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if not base_path.exists():
+        base_path.mkdir(parents=True)
+    else:
+        shutil.rmtree(base_path)
+        base_path.mkdir(parents=True, exist_ok=True)
```

Comment:
it is enough just to remove dir and then create it from scratch

Reply:
done